### DISCUSSION TITLE
open .xml with -no-gui

### DISCUSSION THREAD

**Discussion by grigory707** [cagnulein/qdomyos-zwift#166] [Q&A]
Hello Roberto.

Can you say please is it possible to load xml training program with no gui?

**Comment by cagnulein** [comment]
Yes there is already an option, check the main.cpp

Il dom 7 feb 2021, 13:55 grigory707 <notifications@github.com> ha scritto:

**Reply by grigory707** [reply]
Thank you.
Option "-train". But when i am run
$sudo ./qdomyos-zwift -no-gui -no-logs -train test.xml
"-no-logs" work but "-train" not work. treadmill don't move.

**Comment by cagnulein** [comment]
I'm outside so I can't check the code. You have to press start on your
treadmill to start the program

Il dom 7 feb 2021, 14:18 grigory707 <notifications@github.com> ha scritto:

**Comment by grigory707** [comment]
> I'm outside so I can't check the code.

Thanks!

> You have to press start on your
treadmill to start the program

Its just start with 0.5 speed 0 incline and nothing more.

**Comment by cagnulein** [comment]
Remove the nologs option so you will see what's wrong. Try sorrounding the
file name with "" and of course check if the path is ok

Il dom 7 feb 2021, 14:26 grigory707 <notifications@github.com> ha scritto:
